unix/lib/xs_transport_unix.ml

```diff
@@ -29,8 +29,15 @@ type channel = Lwt_unix.file_descr * Lwt_unix.sockaddr
 let create () =
   let sockaddr = Lwt_unix.ADDR_UNIX(!xenstored_socket) in
   let fd = Lwt_unix.socket Lwt_unix.PF_UNIX Lwt_unix.SOCK_STREAM 0 in
-  lwt () = Lwt_unix.connect fd sockaddr in
+  lwt () =
```

Comment:
This is a funny syntax withot let... I think it's the camlp4 extension... in that case we should not update this code as it is unmaintained